### ISSUE TITLE
Can not handle map with nxstart/nystart/nzstart!=0

### ISSUE THREAD

**Issue by wang3702** [3dem/model-angelo#77]
I tested model-angelo on map EMD-6149 and EMD-3133. The final modeled structure  will be shifted because current pipeline did not handle the nxstart well. You can consider to use this function https://github.com/kiharalab/CryoREAD/blob/main/data_processing/Unify_Map.py before passing the map into further processing

**Comment by jamaliki** [comment]
Hi,

Ah, is this in case the global origin does not point to the correct origin and needs to be shifted by the n[xyz]start?

**Comment by wang3702** [comment]
Yes. I mean n[xyz]start is part of the coordinate system, you should include script to process it. Actually, for some older maps, these fields are more commonly used.

**Comment by jamaliki** [comment]
Interesting, I thought we already handled this in our `load_mrc` function. So this is an issue with EMD-6149? I will take a look.
